docs/testing/sideload-an-office-add-in-on-ipad-and-mac.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Sideload Office Add-ins on iPad and Mac for testing
 description: ''
-ms.date: 12/04/2017
+ms.date: 2/25/2019
```

Comment:
@AlexJerabek The monthly CATS reports will throw errors for any dates that don't have a 2-digit month, 2-digit day, and 4-digit year. Can you please change this date to that format (i.e., 02/25/2019)? Thanks!

Reply:
Fixed. Thanks @kbrandl!